src/graphics/customColorOptions.ts

```diff
@@ -0,0 +1,17 @@
+export default {

```

Comment:
This is probably nicer as a string enum, to help Typescript out.

```
export default enum ColorOption {
  Maroon = 'Maroon',
  ...
}
```

Reply:
Agreed. Although sadly you can't put together "export default enum..." as per:
https://github.com/Microsoft/TypeScript/issues/3320#issuecomment-107241679

I'll do the alternative suggestion instead

Reply:
Ah yeah, same as classes. Separate export is fine too 👍